src/Nethermind/Nethermind.Db/Config/DbConfig.cs

```diff
@@ -86,6 +86,12 @@ public class DbConfig : IDbConfig
         public uint EthRequestsDbWriteBufferNumber { get; set; } = 4;
         public ulong EthRequestsDbBlockCacheSize { get; set; } = 8.MB();
         public bool EthRequestsDbCacheIndexAndFilterBlocks { get; set; } = true;
+        
+        public ulong BloomDbWriteBufferSize { get; set; } = 1.KB();
```

Comment:
why so small?

Reply:
its only for few scalar values, but I wanted it to live in same place as flat files, so we could just drop the folder if needed.